monocle/github/pullrequest.py

```diff
@@ -563,29 +606,6 @@ def extract_pr_objects(pr: Dict) -> List[Union[Change, Event]]:
         return objects
 
 
-class TokenGetter:
```

Comment:
why the class was moved from current location? There was no changes related to it, right?

Reply:
Yes it has been used in the GithubCrawlerArgs dataclass definition.